apps/web/test/lib/handleChildrenEventTypes.test.ts

```diff
@@ -40,8 +39,7 @@ describe("handleChildrenEventTypes", () => {
         currentUserId: 1,
         hashedLink: undefined,
         connectedLink: null,
-        // @ts-expect-error Prisma v5 typings are not yet available
-        prisma: prismaMock,
+        prisma: prismaMock as PrismaClient,
```

Comment:
Can we cast directly on the prismaMock source? So we don't have to repeat this everywhere?